packages/fx-core/src/question/inputs/AddPluginInputs.ts

```diff
@@ -17,10 +17,16 @@ export interface AddPluginInputs extends Inputs {
   "plugin-manifest-path"?: string;
   /** @description Import OpenAPI Description Document */
   "plugin-opeanapi-spec-path"?: string;
+  /** @description OpenAPI Spec Document */
+  "openapi-spec-type"?: "enter-url-or-open-local-file" | "search-api";
```

Comment:
What is these changes for? I believe this file and AddPluginOptions.ts is auto-generated when building. Would you please run `npm run setup` under root folder if the file changes are by mistake?

Reply:
I appears the latest `npm run setup` is bringing these file changes.

Reply:
They seem related to the latest updates to the flow from @SLdragon.